src/libvirtApi/snapshot.js

```diff
@@ -30,9 +30,27 @@ import { call, Enum, timeout } from './helpers.js';
 import { logDebug } from '../helpers.js';
 
 export function snapshotCreate({ connectionName, vmId, name, description, memoryPath, disks, isExternal, storagePools }) {
```

Comment:
Please also drop the now unused properties from this function.

Reply:
Done.